docs/development/release.rst

```diff
@@ -39,50 +56,75 @@ On the day of the release candidate:
    * In the ``download/install`` folder, copy a previous environment file file as ``gammapy-1.0rc1-environment.yml``.
    * Adapt the dependency conda env name and versions as required in this file.
 
-#. Update the ``CITATION.cff`` date and version by running the ``dev/prepare-release.py`` script.
-#. Locally create a new release candidate tag on the v1.0.x, like ``v1.0rc1`` for Gammapy and push. For details see the
-   `Astropy release candidate instructions <https://docs.astropy.org/en/latest/development/maintainers/releasing.html#releasing-the-first-feature-release-candidate>`__.
-#. Once the tag is pushed the docs build and upload to `PyPi <https://pypi.org/>`__ should be triggered automatically.
-#. Once the docs build has succeeded find the ``tutorials_jupyter.zip`` file for the release candidate
-   in the `gammapy-docs repo <https://github.com/gammapy/gammapy-docs>`__ and adapt the ``download/index.json`` to point to it.
-#. Update the entry for the release candidate in the `Gammapy release calendar <https://github.com/gammapy/gammapy/wiki/Release-Calendar>`__.
-#. Create a testing page like `Gammapy v1.0rc testing <https://github.com/gammapy/gammapy/wiki/Gammapy-v1.0rc-testing>`__.
+#. Switch to the correct branch and update the ``CITATION.cff`` date and version by running the
+   ``dev/prepare-release.py`` script::
+
+    git checkout v1.0.x
+    python ./dev/prepare-release.py --release v1.0rc1
+
+#. Push the branch back to GitHub::
+
+    git push upstream v1.0.x
+
+#. Locally create a new release candidate tag on the ``v1.0``, like ``v1.0rc1`` for Gammapy and push::
```

Comment:
```suggestion
#. Locally create a new release candidate tag on the ``v1.0.x`` branch, like ``v1.0rc1`` for Gammapy and push::
```

Reply:
The `x` would imply a bug fix release. So I don't think we want to include that here

Reply:
No, `v1.0.x` or `v1.3.x` is the name of the branch. Maybe we can clarify this.

Reply:
Implicitly v1.0 is v1.0.0, so the x is always there. 
In their latest APE, astropy decided to use X.0.0 for the major release as it is clearer this way. For now, we don't

Reply:
Ahh okay, I follow now. 
Thanks for the clarification